document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -314,10 +348,13 @@ private String prepareParameterizedStringForList(
     return "(" + collect + ")";
   }
 
-  private StringBuilder prepareFieldAccessorExpr(String fieldName) {
+  private StringBuilder prepareFieldAccessorExpr(String fieldName, String aliasPrefix) {
```

Comment:
Can you please add the `@Nullable` notation for `aliasPrefix`?

Reply:
Done